beater/config.go

```diff
@@ -66,23 +65,23 @@ func (s *Sourcemapping) isSetup() bool {
 	return s != nil && (s.esConfig != nil)
 }
 
-func (c *FrontendConfig) SmapAccessor() utility.SmapAccessor {
+func (c *FrontendConfig) SmapMapper() sourcemap.Mapper {
 	smap := c.Sourcemapping
 	if c.isEnabled() && smap.isSetup() {
-		if smap.accessor == nil {
-			smapConfig := utility.SmapConfig{
+		if smap.mapper == nil {
```

Comment:
While we're refactoring, how about flipping the conditional, `return nil` early?